src/main/java/org/radarcns/management/config/SecurityConfiguration.java

```diff
@@ -34,6 +37,9 @@
 
     private final UserDetailsService userDetailsService;
```

Comment:
I think a good practice is to either do constructor based DI or using `@AutoWired`. Using solely `@AutoWired` is preferred and it eases testing too.

Reply:
Indeed, took the liberty change this in the other configuration classes as well